docs/modules/development/pages/rest/snmp_configuration.adoc

```diff
@@ -64,8 +64,8 @@ All parameters can be set regardless the version.
 [options="header"]
```

Comment:
In line 62, there is a missing word: 

All parameters can be set regardless of the version.